Add hide and show to submit purchase button

ptavarez opened the pull request:
Submit Purchase Bug
 Initial issue:
   - Submit purchase button will show when cart is empty
   - This made the button inactive
  Solution
   - Add hide function that hides the button if cart is empty within the
   getActiveCartFailure function
   - Add show function that shows the button when item is added to cart
   in the addToCartSuccess function
(Closes issue #30)

Logos
 Favicon logo
   - Add link tag that adds a favicon logo to our page
 Stripe checkout image
  - Add custom image as our checkout logo

Commit:
Add hide and show to submit purchase button
 Initial issue:
   - Submit purchase button will show when cart is empty
   - This made the button inactive
 Solution
   - Add hide function that hides the button if cart is empty within the
   getActiveCartFailure function
   - Add show function that shows the button when item is added to cart
   in the addToCartSuccess function

Commit:
Add favicon
- Add link tag in html head for a favicon image

ismurray reviewed (changes requested):
Need to add $('#submit-purchase-stripe').show() to signInSuccess so that if a customer with an empty cart logs out, and then a customer with a cart logs in, the button won't still be hidden

Commit:
Replace stripe checkout logo
- Replace stripe checkout default image for a custom logo

Commit:
Fix submit purchase bug
- Add $('#submit-purchase-stripe').show() to sign in to prevent missing
submit purchase button
